tagged-commit.mjs

```diff
@@ -1,13 +1,25 @@
 import pkg from './package.json'
 import generateBuildNumber from './node_modules/alpheios-node-build/dist/support/build-number.mjs'
 import { execFileSync, execSync } from 'child_process'
+import branch from 'git-branch'
```

Comment:
We're importing branch package in `node-build` so maybe it's better to avoid dependency on `branch` here and update node-build to return the branch name? We could also move a functionality of extracting the base version into `node-build` as it will be the same across several `tagged-commit.mjs` scripts. Then once we decide to change its logic, we could change it in `node-build` and it will be updated across all packages that use it. On the other hand, it is a small piece of code. What do you think?

Reply:
yeah, I started to do it in node-build, but then, decided against it. It probably is better in node-build. I think I'd like though to make sure this actually works with a full travis build and refactor it after.

Reply:
OK, that's fine with me